components/script/dom/element.rs

```diff
@@ -308,6 +309,7 @@ impl Element {
             state: Cell::new(state),
             selector_flags: Cell::new(ElementSelectorFlags::empty()),
             rare_data: Default::default(),
+            cryptographic_nonce: DomRefCell::new("".to_owned()),
```

Comment:
We should consider putting this in the ElementRareData to minimize the memory usage increase to only elements that have a nonce attribute.